src/handler.lua

```diff
@@ -21,10 +21,47 @@ local plugin = {
     VERSION = "0.1",
 }
 
+DB = {}
```

Comment:
```suggestion
_G.DB = {} -- required if using 4DaysORMAdapter
```
I think we should have some context here.